CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing
+
+Thank you for your interest in contributing to the Devfile Registry! We welcome your additions to this project.
+
+## Code of Conduct
+
+Before contributing to this repository for the first time, please review our project's [Code of Conduct](https://github.com/devfile/api/blob/main/CODE_OF_CONDUCT.md)
+
+## How to Contribute:
+
+### Issues
+
+If you spot a problem with the devfile registry, [search if an issue already exists](https://github.com/devfile/api/issues). If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/devfile/api/issues/new/choose).
+
+You can tag Devfile Registry related issues with the `/area registry` text in your issue.
+
+### Development
+
+#### Repository Format
+
+The `registry-support` repository for the devfile registry is a mono-repo of sorts and there are a number of individual components contained within.
+
+- [Devfile Registry Build Tools](./build-tools) - Tools for building devfile registries into container images.
+- [Devfile Registry Helm Chart](./deploy/chart/devfile-registry) - Helm chart for deploying the devfile registry on to Kubernetes.
+- [Index Generator](./index/generator) - The index generator tool for generation index.json files from registry data
+- [Index Server](./index/server) - The index server, one of two servers that the devfile registry runs
+- [OCI Registry Server](./oci-registry) - 
```

Comment:
Missing the description for the OCI Registry Server.

Reply:
Fixed.